cmd/queue/main.go

```diff
@@ -133,33 +119,39 @@ var (
 		stats.UnitMilliseconds)
 )
 
-func initEnv() {
-	containerConcurrency = util.MustParseIntEnvOrFatal("CONTAINER_CONCURRENCY", logger)
-	queueServingPort = util.MustParseIntEnvOrFatal("QUEUE_SERVING_PORT", logger)
-	revisionTimeoutSeconds = util.MustParseIntEnvOrFatal("REVISION_TIMEOUT_SECONDS", logger)
-	servingConfig = util.GetRequiredEnvOrFatal("SERVING_CONFIGURATION", logger)
-	servingNamespace = util.GetRequiredEnvOrFatal("SERVING_NAMESPACE", logger)
-	servingPodIP = util.GetRequiredEnvOrFatal("SERVING_POD_IP", logger)
-	servingPodName = util.GetRequiredEnvOrFatal("SERVING_POD", logger)
-	servingRevision = util.GetRequiredEnvOrFatal("SERVING_REVISION", logger)
-	servingService = os.Getenv("SERVING_SERVICE") // KService is optional
-	userTargetPort = util.MustParseIntEnvOrFatal("USER_PORT", logger)
-	userTargetAddress = "127.0.0.1:" + strconv.Itoa(userTargetPort)
-	userContainerName = util.GetRequiredEnvOrFatal("USER_CONTAINER_NAME", logger)
-
-	enableVarLogCollection, _ = strconv.ParseBool(os.Getenv("ENABLE_VAR_LOG_COLLECTION")) // Optional, default is false
-	varLogVolumeName = os.Getenv("VAR_LOG_VOLUME_NAME")
-	if varLogVolumeName == "" && enableVarLogCollection {
+type config struct {
+	ContainerConcurrency         int    `split_words:"true" required:"true"`
```

Comment:
I assume `split_words` is a way of deriving the name in place of the explicit form? https://github.com/knative/eventing/blob/a33f53317155e4769505f55c8185324a71bfbe42/test/test_images/heartbeats/main.go#L54

cc @n3wscott

Reply:
yes.
https://github.com/kelseyhightower/envconfig#struct-tag-support
```Envconfig has automatic support for CamelCased struct elements when the split_words:"true" tag is supplied```